packages/create-neutrino-project/commands/init/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
No strict directives.

Reply:
Out of curiosity, what is the reasoning behind removing the strict directives :)